tests/Makefile.am

```
@@ -18,7 +18,7 @@ AM_TESTSUITE_SUMMARY_HEADER = ' for $(PACKAGE_STRING) System Tests'
 LOG_DRIVER = $(SHELL) $(top_srcdir)/forked-test-driver
```

```suggestion
SH_LOG_DRIVER = $(SHELL) $(top_srcdir)/forked-test-driver
```
We need to update this due to the TEST_EXTENSIONS change.